scipeds/data/completions.py

```diff
@@ -142,6 +148,7 @@ def format_join(cols: list[str], var_name: str, table_name: str) -> tuple[str, s
             institution_name=f"{INSTITUTIONS_TABLE}.institution_name,"
             if "unitid" in total_cols
             else "",
```

Comment:
I know this isn't in this PR, but is this missing a keyword arg?

Reply:
I don't think so? Are you referring to the `GROUP_FIELDS_QUERY` or the _format_query function?